samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/index.js

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
+import ReactDOM from 'react-dom';
 import './index.css';
 import { AppRoute } from './router/router';
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
-  <React.StrictMode>
-    <AppRoute />
-  </React.StrictMode>
-);
+ReactDOM.render(
+    <React.StrictMode>
+        <AppRoute />
+    </React.StrictMode>, document.getElementById('root')
```

Comment:
Why did you remove configureController.cs file ?

Reply:
Migration to React doesn't required controller files